analytics/src/analytics/cli.py

```diff
@@ -162,24 +163,12 @@ def export_json_to_database(
     # Get the database engine and establish a connection
     engine = db.get_db()
 
-    # get data and load from JSON
-    deliverable_data = DeliverableTasks.load_from_json_files(
-        sprint_file=sprint_file,
-        issue_file=issue_file,
-    )
-
     # Load data from the sprint board
     sprint_data = SprintBoard.load_from_json_files(
         sprint_file=sprint_file,
         issue_file=issue_file,
     )
 
-    deliverable_data.to_sql(
-        output_table="github_project_data",
-        engine=engine,
-        replace_table=True,
-    )  # replace_table=True is the default
```

Comment:
Basically, the `DeliverableTasks` dataset is a psuedo-interface that expects a certain shape of data to be exported from GitHub. When we switched to using sub-issues instead of a `deliverable` label in our GitHub projects, we broke that "contract" with the `DeliverableTasks` interface.

The short term fix for this bug was to remove the invocation of that code from the import step so we aren't calling the `DeliverableTasks` and instead we're only loading `SprintBoard` data into the db. The long-term fix is to replace these "old" interfaces with a new interface `GitHubIssues` that has more direct validation (the focus of #2539), then to update all previous references to the old datasets with GitHubIssues`